tests/queries/0_stateless/00816_long_concurrent_alter_column.sh

```diff
@@ -59,6 +59,7 @@ wait
 
 echo "DROP TABLE concurrent_alter_column" | ${CLICKHOUSE_CLIENT}
 
+sleep 7
```

Comment:
Does not look like the fix.

Reply:
> Does not look like the fix.

`git log` and linked PRs said that this test was supposed to check for deadlocks, while immediately checking for running queries occasionally showed just one query lagging behind.

What solution do you suggest? To poll until it's gone with shorter delay?

Reply:
DROP TABLE query takes exclusive lock and will wait for all running queries on the same table.

Reply:
No ALTER TABLE queries can execute after DROP TABLE.

Reply:
We should investigate the issue further, I will remove the "fake" fix.